adapt-service-without-webapi/README.md

```diff
@@ -0,0 +1,25 @@
+# Custom Service Adatper with Knot.x
+
+In this tutorial we create a page: [http://localhost:8092/content/local/books.html](http://localhost:8092/content/local/books.html) 
```

Comment:
I'd rather leave it with `/content/local/books` - I want to avoid additional configuration for repository. This is not a case of this tutorial.